cruise-control-client/setup.py

```diff
@@ -11,7 +11,7 @@
 
 setuptools.setup(
     name='cruise-control-client',
-    version='0.1.5',
+    version='1.0.0',
```

Comment:
Just out of curiosity, why is this a major version bump up?

Reply:
Solely due to the deletion of the `get_responder` function in `cccli.py`, which still works after this PR, just makes use of deprecated classes.

Given that, it is probably better to add this function back in and include a deprecation warning, and relegate to minor version bump.
0a2c3a8